core/src/services/gcs/backend.rs

```diff
@@ -501,6 +503,52 @@ impl Accessor for GcsBackend {
             GcsPager::new(self.core.clone(), path, "", args.limit()),
         ))
     }
+    async fn batch(&self, args: OpBatch) -> Result<RpBatch> {
+        let ops = args.into_operation();
+        if ops.len() > 99 {
```

Comment:
This should be `> 100`?

Reply:
>You should not include more than 100 calls in a single batch request. 

Not sure whether batch request itself is included, so in case, 99 means 99 delete calls and 1 batch header.

Reply:
I guess it's 100.